### DISCUSSION TITLE
Hosting a Uno WebAssembly App on IIS server (Not using Azure Static Web Apps)

### DISCUSSION THREAD

**Discussion by hbraasch** [unoplatform/uno#19140] [General]
Can anyone please assist on how to host a Uno generated WebAssembly app on a IIS Server?

- What part of the `...\bin\Release\net8.0-browserwasm\dist`  folders/files needs to be placed in the server folder?
- How must the server be configured? What additional stuff needs to be installed on the server e.g. URL Rewrite?
- Does the generated  `web.config` needs to be to changed to be accepted by the server?
- And so on.

I'm using SmarterAsp for hosting. I created a subsite, then FTP'ed all files in the [dist] folder to the subsite's folder. Stop/Started the server, then browsed to it. It does not work out of the box. Hitting F12 on the browser shows lots of issues.

I had some success hosting it as an Azure Static App, but ran into other troubles and for now gave up using that approach.

I'm at a loss. Help shall be much appreciated.

Thanks

**Comment by jeromelaban** [comment]
Thanks for the question!

You can find some steps on [how to host an app in our docs](https://platform.uno/docs/articles/how-to-host-a-webassembly-app.html).
